src/index.js

```diff
@@ -75,6 +80,17 @@ export default class SaucelabsConnector {
         if (timeout)
             initParams.maxDuration = timeout;
 
+
+        // NOTE: if starts IE 11, the option "Display intranet sites in Compatibility View" should be disabled.
+        // We do this via 'prerun' parameter, which should run our script on Sauce Labs side, before the browser starts.
+        if (browser.browserName.toLowerCase() === 'internet explorer' && /11(\.\d*)?$/.test(browser.version)) {
+            // NOTE: We should upload the script if it's unavailable on the sauce-storage.

```

Comment:
He means we should upload the file there no this file in the storage (we haven't uploaded it yet)